docs/src/main/asciidoc/podman.adoc

```diff
@@ -8,73 +8,37 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 
-=== macOS
+See https://podman-desktop.io/downloads/ for the latest version of Podman Desktop or pick the version that suits your operating system from the list below:
 
-Containers are really Linux.
-As such, Linux containers cannot run natively on macOS or Windows.
-Therefore, the containers must run in a Linux virtual machine (VM), and a Podman client interacts with that VM.
-A native hypervisor subsystem and virtualization software is used to run the Linux VM on the OS, and then containers are run within this VM.
-In Podman, this is known as the Podman machine, and it is built into the tool.
+- https://podman-desktop.io/linux/[Linux]
+- https://podman-desktop.io/macos/[MacOS]
+- https://podman-desktop.io/windows/[Windows]
 
-macOS users can install Podman through https://brew.sh/[Homebrew].
-Once you have set up `brew`, you can use the `brew install` command to install Podman and `docker-compose`:
 
-[source,bash]
-----
-brew install podman
-brew install docker-compose
-podman machine init -v $HOME:$HOME
-PODMAN_VERSION=`podman -v | sed 's/[a-zA-Z ]*//'`
-sudo /opt/homebrew/Cellar/podman/$PODMAN_VERSION/bin/podman-mac-helper install
-podman machine set --rootful
-podman machine start
-alias docker='podman'
-----
+== Known/Past issues with Podman
 
-If you're using Podman 4.1 or higher, you don't need the `-v $HOME:$HOME` volume mount.
+=== Testcontainers privileges
 
-If you're using Mac M1, an extra step is required to https://edofic.com/posts/2021-09-12-podman-m1-amd64[make AMD64 images work]:
+Edit `~/.testcontainers.properties` and add the following line
 
-[source,bash]
+[source,properties]
 ----
-podman machine ssh
-sudo -i
-rpm-ostree install qemu-user-static
-systemctl reboot
+ryuk.container.privileged=true
 ----
 
-Once the virtual machine restarts, you should be good to run dev services.
-
-For more details, please see
-
-- the https://podman.io/getting-started/installation#macos[official Podman documentation]
-- article about https://www.redhat.com/sysadmin/replace-docker-podman-macos[running Podman on Mac]
-- https://xphyr.net/post/podman_on_osx/[another article], with good guidance on `--rootful` and mounting volumes
-- article about https://edofic.com/posts/2021-09-12-podman-m1-amd64[running AMD64 images with Podman on Mac M1]
-
-=== Windows
-
-Please see the https://github.com/containers/podman/blob/main/docs/tutorials/podman-for-windows.md[Podman for Windows guide] for setup and usage instructions.
-
-Before starting the Podman machine, set it to prefer rootful container execution:
+Alternatively, you can disable ryuk:
 
 [source,bash]
 ----
-podman machine set --rootful
+export TESTCONTAINERS_RYUK_DISABLED=true #not recommended - see above!
```

Comment:
Two comments - I wonder if we should maybe not continue to recommend this? The podman folks strongly recommend that we don't.

Also, the 'see above' doesn't make sense anymore because the explanatory links have been removed. It would need to be 'see below'.

Reply:
this is no longer required since test containers now sets privileged to true by default.